examples/src/Example1.re

```diff
@@ -24,15 +24,13 @@ let make = (~options, _children) => {
   render: self =>
     <Select
       options
-      value=?(
-        Js.Option.map(
-          (. b) => Select.Option.Val(b),
-          self.state.selectedItem,
-        )
-      )
-      onChange=(selected => self.send(Change(selected)))
-      arrowRenderer=((_) => <div> (ReasonReact.stringToElement("+")) </div>)
-      filterOptions=(Func((~options, ~filter as _filter) => options))
-      placeholder=(Str("Select something.."))
+      value=?{
+        self.state.selectedItem
+        |> Js.Option.map((. b) => Select.Option.Val(b))
```

Comment:
Reason community seems to prefer `Belt.Option` now. I spent a minute trying to change to this, but couldn't figure it out so moved on.

Reply:
what is the error is?

Generally speaking smth like this should work
```
open Belt;
...
Option.map(self.state.selectedItem, item => Select.Option.Val(item))
```